jupyter_book/commands/__init__.py

```diff
@@ -36,44 +36,36 @@ def main():
     pass
 
 
-BUILDER_OPTIONS = ["html", "pdfhtml", "linkcheck", "latex", "pdflatex"]
+BUILDER_OPTS = {
+    "html": "html",
+    "pdfhtml": "singlehtml",
+    "latex": "latex",
+    "pdflatex": "latex",
+}
 
 
 @main.command()
-@click.argument("path-book")
+@click.argument("path-book", type=click.Path(exists=True, file_okay=False))
```

Comment:
do you know if there's some way to make this default to `.`?

Reply:
I would be shocked if there isn't.  I'll take a look.